class-decent-comment.php

```diff
@@ -187,6 +187,11 @@ function query( $query_vars ) {
 		if ( ! empty( $post_id ) ) {
 			$where .= $wpdb->prepare( ' AND comment_post_ID = %d', $post_id );
 		}
+    else {
+      //if no post_id is specified, exclude current post id, since theme should show it elsewhere:
```

Comment:
I suspect that the standard behaviour is probably a minor problem for most users of the plugin and so this should be included as an option at least.
Normally the comments for the current page are in a separate block which can be replied to, whereas the comments from decent comments you would normally have to go back to the referring page to reply.
If the current page comments are also in the decent comments block that's just a little confusing.